contracts/mocks/MockSTETHToken.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import {ERC20Upgradeable} from "../packages/oz/upgradeability/ERC20Upgradeable.sol";
+
+contract MockSTETHToken is ERC20Upgradeable {
```

Comment:
This is a mock for stETH, but we need a wstETH token, which has doesn't have getPooledEthByShares.